src/syntax-highlight/language-ext.js

```diff
@@ -69,5 +69,6 @@ module.exports = {
 
     // Universally named files
     jenkinsfile: 'language-groovy',
-    vagrantfile: 'language-ruby'
+    vagrantfile: 'language-ruby',
+    rakefile: 'language-ruby'
```

Comment:
I would like to keep them alphabetically ordered, could you put `rakefile` between `jenkinsfile` and `vagrantfile` please?

Reply:
Yeah I totally agree, that makes sense.